### PR TITLE
stop hard coding connection secret namespaces

### PR DESCRIPTION
**Pull request by jbw976**
set providerconfig.helm.release.io to look for secrets in the parent XR namespace

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
LGTM!

File: cluster/eks/composition.yaml [hunk 1]
```diff
@@ -174,11 +174,12 @@ spec:
           credentials:
             source: Secret
             secretRef:
-              namespace: crossplane-system
```
Comment (prasek) [inline cluster/eks/composition.yaml:177]: @jbw976 is this implicit namespace behavior documented somewhere? I looked on crossplane.io/docs but didn't find anything.

**Review by prasek — APPROVED** [review]
Looks good, plus one minor comment re: docs for implicit namespace inheritance behavior.

File: cluster/eks/composition.yaml (hunk 1 above)
Comment (jbw976) [reply cluster/eks/composition.yaml:177]: can you clarify a bit more what you mean by "implicit namespace behavior" @prasek? the namespace is explicitly being set below in the patches, but perhaps I don't understand fully what you are implying

Comment (prasek) [reply cluster/eks/composition.yaml:177]: my bad, I misread the patches while multi-tasking. 👍

**Pull request merged**